tests/test_contrib.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError
+
+from marshmallow_utils.fields import contrib
+
+
+def serialize_foo(obj, context):
+    return {'serialize_args': {'obj': 1, 'context': 2}}
+
+
+def deserialize_foo(value, context, data):
+    return {'deserialize_args': {
+        'value': 3, 'context': 4, 'data': 5}}
+
+
+class MySchema(Schema):
+    f = contrib.Function(serialize_foo, deserialize_foo)
```

Comment:
And remove the initial package

```suggestion
    f = Function(serialize_foo, deserialize_foo)
```